uisdk/src/main/java/com/karhoo/uisdk/util/LettersOnlyFilter.kt

```diff
@@ -0,0 +1,21 @@
+package com.karhoo.uisdk.util
+
+import android.text.InputFilter
+import android.text.Spanned
+
+class LettersOnlyFilter : InputFilter {
+    override fun filter(
+        source: CharSequence?,
+        start: Int,
+        end: Int,
+        dest: Spanned?,
+        dstart: Int,
+        dend: Int
+                       ): CharSequence {
+        if (source == null) return ""
+        if (source == "") { // for backspace
+            return source
+        }
+        return Regex("[^A-Za-z ]").replace(source, "")
```

Comment:
Not sure if it works with Arabic or latin names 

the API expectation is 
https://github.com/karhoo/api-v1-bookings/blob/master/docs/swagger/bookings.yaml#L1544

they just exclude some special characters

Reply:
Ok, i will change it, but also the requirement was to exclude numbers from the name. In my opinion we shouldn't even do this filter check, i think setting the `android:inputType="textPersonName|textCapWords"` to the inputtext should be enough for normal use cases and we should not interfere with it